lib/index.js

```diff
@@ -51,8 +57,8 @@ class BaseAsset {
 
     get asset_id() { return this._asset_id; }
 
-    set_title(value) { this._title = value; }
-    set_synopsis(value) { this._synopsis = value; }
+    set_title(value) { this._title = _sanitize_text(value); }
```

Comment:
@tchx84 it's a bit tricky, since we may want to sanitize text differently for different fields. i.e. newlines aren't desired in titles, but could be desired elsewhere